components/Storyblok/SbQuote.tsx

```diff
@@ -58,7 +58,7 @@ export const SbQuote = ({
       isLargeBody={isLargeBody}
       source={source}
       barColor={paletteAccentColors[value]}
-      quoteOnRight={quoteOnRight}
+      barOnLeft={quoteOnRight}
```

Comment:
Why not change the Storyblok field name too?

Reply:
Because I want to keep the backward compatibility. The quote is used in many places so I'd rather not have to manually go in and change them all out in Storyblok if I rename the field.

Reply:
Eh? Wouldn't you just have to change the one Storyblok blok (Quote)? Changing the field name keeps the data as far as I recall.

Reply:
It does keeps the data until a user opens up a story with a quote on it. Then it will empty out the selection that was picked due to the field name change. I noticed that on this site, they do go back and update old stories a lot, and they might not know to check existing quotes whether the option they have selected has changed 😬

Reply:
Wait what? That seems like a very bad bug in Storyblok's UI.

Reply:
We almost never update a field name after a component has been deployed to live, so I'm relying on memory of how it once was. Next time I can test this out before we deploy a new component 😂 For now, I'd rather not risk this one.